File: tutorials/testing/python/world_map.py
from bridges.bridges import *
from bridges.data_src_dependent import *
from bridges.dl_element import *
from bridges.world_map import *

def main():
    # create the Bridges object, set credentials
    bridges = Bridges(YOUR_ASSSIGNMENT_NUMBER, "YOUR_USER_ID", "YOUR_API_KEY")

    # set title, description
    bridges.set_title("World Map Using BRIDGES")

    #country_data = get_world_map_data(['Australia', 'Germany', 'France'])
    country_data = get_world_map_data()
    wm = WorldMap(country_data)
    
    #visualizes world map
    bridges.set_data_structure(wm)
    bridges.set_json_flag(True)
    bridges.visualize()

if __name__ == '__main__':
    main()

File: tutorials/testing/java/world_map.java
import bridges.connect.Bridges;
import bridges.connect.DataSource;
import bridges.base.WorldMap;
import bridges.data_src_dependent.Country;


import java.util.ArrayList;
import java.lang.String;


public class world_map {
	public static void main(String[] args) throws Exception {

		// create Bridges object
		Bridges bridges = new Bridges (YOUR_ASSSIGNMENT_NUMBER, "YOUR_USER_ID", "YOUR_API_KEY");
		// set title, description
		bridges.setTitle("Using World Map");

		DataSource ds = bridges.getDataSource();
		
		// get Country data
		String[] countries = {"Germany", "France"};
		ArrayList<Country> country_data = ds.getWorldMapData();
		
		// create a USMap object with the map data
		WorldMap wm = new WorldMap(country_data);

		// set data structure
		bridges.setDataStructure(wm);

		// visualize
		bridges.visualize();
	}
}
